app/js/module/dom.js

```diff
@@ -89,5 +89,15 @@ asciidoctor.browser.dom = ((document) => {
     return div.innerHTML;
   };
 
+  /**
+   * Set the appropriate viewport <meta> tag
+   */
+  module.setViewport = () => {
```

Comment:
Since we are using an existing HTML document, should we remove the existing viewport meta (if it exists) ?

@mojavelinux What do you think ?

Reply:
Yes, I think so. The extension owns the page, so it should do with it whatever it feels necessary.